go.mod

```diff
@@ -31,7 +31,7 @@ require (
 	github.com/openshift/api v0.0.0-20210105115604-44119421ec6b
 	github.com/pborman/uuid v1.2.0
 	github.com/pkg/errors v0.9.1
-	github.com/portworx/px-backup-api v1.2.2-0.20220801043531-88280e08adeb
+	github.com/portworx/px-backup-api v1.2.2-0.20220822053657-49308ab319f1
```

Comment:
is it the latest one? do we have 2.3 or 2.3?

Reply:
This the master of px-backup-api (latest)